### PR TITLE
[Add-ins] (AppDomains) Note browser behavior

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[236a451](https://github.com/OfficeDev/outlook-dev-docs/pull/597/commits/236a451d84f729b8142db598992f280877b0d9c3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/manifests.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-browsers/docs/add-ins/manifests.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C29%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201905292243406568-597%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=HUFswYlkWZrhv9ny9YbKxKMvxZAF0EubBuBgsWeqGsU%3D&st=2019-05-29T22%3A39%3A34Z&se=2019-06-29T22%3A44%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/manifests.md [hunk 1]
```diff
@@ -367,6 +367,16 @@ The following example specifies a domain  `https://www.contoso2.com` as a second
 
 App domains are also necessary to enable cookie sharing between the pop-out window and the add-in running in the rich client.
 
+The following table describes the browser behavior for URLs not belonging to the add-in's default domain.
```
Comment (kbrandl) [inline docs/add-ins/manifests.md]:
```suggestion
The following table describes browser behavior when your add-in attempts to navigate to a URL outside of the add-in's default domain.
```

File: docs/add-ins/manifests.md [hunk 2]
```diff
@@ -367,6 +367,16 @@ The following example specifies a domain  `https://www.contoso2.com` as a second
...
+
+|Outlook client|Domain defined<br>in AppDomains?|Browser behavior|
+|---|---|---|
+|All clients|Yes|Link opens in add-in task pane.|
+|Outlook 2013 on Windows<br>Outlook 2016 on Windows (one-time purchase)|No|Link opens in Internet Explorer 11.|
```
Comment (kbrandl) [inline docs/add-ins/manifests.md]: Is the text "(one-time purchase)" intended to apply to both "Outlook 2013 on Windows" and "Outlook 2016 on Windows" ?  If yes, then I'd suggest adding the text "(one-time purchase)" after "Outlook 2013 on Windows" as well.

Comment (ElizabethSamuel-MSFT) [reply docs/add-ins/manifests.md]: No, it is not.

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[957fed9](https://github.com/OfficeDev/outlook-dev-docs/pull/597/commits/957fed97f6e8edc5ff933a051cad86a1ffd65f10)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/manifests.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-browsers/docs/add-ins/manifests.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C30%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201905301849178087-597%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=mw%2BOeg9y4AcQ0Ufqd0fGOAlaqqkaA%2FIXaRaPh7Ad72E%3D&st=2019-05-30T18%3A45%3A15Z&se=2019-06-30T18%3A50%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[50692cc](https://github.com/OfficeDev/outlook-dev-docs/pull/597/commits/50692cce5b1073662c9fd1ecc199f89d34536b2d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/manifests.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-browsers/docs/add-ins/manifests.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C30%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201905301850503471-597%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=zBBLNHGQF8NNVS%2F3gHv%2BmAuYXR7Mk4965nEgp2A3W0Y%3D&st=2019-05-30T18%3A46%3A46Z&se=2019-06-30T18%3A51%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by ElizabethSamuel-MSFT** [comment]
@kbrandl Thanks for reviewing!

**Pull request merged**